### ISSUE TITLE
bluetooth

### ISSUE THREAD

**Issue by Dmole** [koalazak/dorita980#140]
Has anyone found a wayl to connect the Roomba to wifi via bluetooth or other means?

**Comment by cs-453453** [comment]
No, I think only way to connect it to WiFi is its WiFi ap and the robots app
